### DISCUSSION TITLE
Comic books support

### DISCUSSION THREAD

**Discussion by gewoondemax** [advplyr/audiobookshelf#1895] [Ideas]
I added CBR files and it works okay. One idea will there be some navigation, or enhancements to enlarge (zoom and or full screen), pages or make spreads this would be awesome!

Thank you for the great work!
The best bookshelf ever 👍

**Comment by advplyr** [comment]
Hey, yes this is planned

**Comment by gewoondemax** [comment]
Awesome cannot wait 🥳🥳🥳

**Comment by Jdiesel87** [comment]
I didn't know anything about digital comics, metadata sources, organizational structure or other naming conventions until a few weeks ago when I dipped my toes into it. I tired Kavita and Komga and settle with Komga because it supported folders better.

If you can do to comics what you've done with audiobooks and fill that much needed gap it would be huge. Komga works but it has no way to scrape metadata and has no ebook or magazine support. I ended up using Mylar to convert my comics from cbr to cbz and tag them with metadata from ComicVine. I believe the metadata format is equivalent to the now defunct ComicRack format. I hope some more knowledgeable folks and step in and provide feedback and contribute code to give comics and magazines some love.

**Reply by Jakerfv** [reply]
Agreed. There is just no good comic solution on the market. It's honestly shocking that a basic feature set (resuming where you left off from an online reading position downloaded from a server, on your downloaded, offline copy when it was last connected to your server) is not apparent on literally every single app. Believe me, it took me years to find anything. The only app that does this is Bookfusion and Bookfusion is a subscription model that only just recently raised the file size to 1 and 2 gigabytes for current plan users and unless you have a legacy plan, you don't have access to a storage size above 100 gigs for your library which is hosted on their servers. I'm sorry but those gigabyte limits per file size might make sense to you but I am not trying to upload Superman 1-1000 on your server, I'm trying to upload them into big chunks and even 1 gig is just too small. I hear they will let you host on your own server which sounds great in the future but for now, I'm just paying on a legacy plan I don't use at all because they just have these restrictions that require tons of dev time to recode under the hood from what I've been told by their people directly.

Komga, Kavita, and Ubooquity are self-hosted so there is no file size limit, great! But they have no parity with their respective apps (which are usually third-party using OPDS which doesn't sync up downloaded files with the server's progress, boo). And anything besides these is a mobile app only, (with the exception of FBReader, which just re-launched their PC version) with no server functionality to call home to and download books from on the go. Jellyfin's might work now, I tried it a long time ago when it was in a pre-release stage (Plex/Emby alternative for movies/shows).

Meanwhile, having audiobookshelf potentially work with comics in the exact same ways for the user that it has for audiobooks? Oh my god, that would be amazing!
